README.md

```diff
@@ -54,6 +54,7 @@ Please note that `@next` will only point to pre-releases; to get the latest stab
 <p style="display: flex; justify-content: center;">
   <a data-ga-event-category="sponsor" data-ga-event-action="logo" data-ga-event-label="octopus" href="https://octopus.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank" style="margin-right: 16px;"><img height="128" width="128" src="https://avatars3.githubusercontent.com/u/1287123?s=256" alt="octopus" title="Repeatable, reliable deployments" loading="lazy" /></a>
   <a data-ga-event-category="sponsor" data-ga-event-action="logo" data-ga-event-label="doit-intl" href="https://www.doit-intl.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank" style="margin-right: 16px;"><img height="128" width="128" src="https://avatars3.githubusercontent.com/u/8424863?s=256" alt="doit-intl" title="Management Platform for Google Cloud and AWS" loading="lazy" /></a>
+  <a data-ga-event-category="sponsor" data-ga-event-action="logo" data-ga-event-label="aptugo" href="https://www.aptugo.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank" style="margin-right: 16px;"><img height="128" width="128" src="/static/sponsors/aptugo.svg" alt="aptugo" title="Augmented Software Development Platform" loading="lazy" /></a>
```

Comment:
@oliviertassinari Hey, how can I upload this new sponsor's logo to the avatar s3 bucket as done with other logos?

Reply:
It is broken now for sure:
![image](https://user-images.githubusercontent.com/4512430/135593772-cded3087-dcf6-4e8b-9d88-1141d36aff8e.png)

Reply:
I assume we should use this https://github.com/AptugosTeam public profile image.

Reply:
@oliviertassinari Their profile picture is here: https://avatars.githubusercontent.com/u/72868791?v=4

It's cropped a bit, and so it looks like this.

![Screenshot 2021-10-01 at 11 07 55](https://user-images.githubusercontent.com/32841130/135602949-a0436d93-b6a9-4cbc-b620-092ccde2c174.png)

Reply:
I found this: https://www.aptugo.com/img/favicon.png

This is not cropped. I will use this.

Reply:
A favicon is meant to render here: ⬇️

<img width="73" alt="Screenshot 2021-10-01 at 21 54 20" src="https://user-images.githubusercontent.com/3165635/135678867-12468508-9808-493a-b7a3-84072fef064e.png">
 
so I don't think it would match the context we use it in. I would personally have used their official GitHub's avatar:  https://avatars.githubusercontent.com/u/73115503

<img src="https://avatars.githubusercontent.com/u/73115503" width="80" />

and let them know that they need to update it. This way, we won't have to change the link if they change their logo.

Reply:
@oliviertassinari I saw in their reply that they want to use the favicon. So, I will merge this now! :)